### PR TITLE
cincinnati: make dockercfg secrets available to cargo_test

### PR DESCRIPTION
**Pull request by steveej**
Depends on https://github.com/openshift/cincinnati/pull/309.

### CODE REVIEW

**Comment by steveeJ** [comment]
https://github.com/openshift/cincinnati/pull/309 is merged now so this should pass.

/retest

**Comment by steveeJ** [comment]
This is ready for final review :raised_hands: @vrutkovs PTAL

File: ci-operator/config/openshift/cincinnati/openshift-cincinnati-master.yaml [hunk 1]
```diff
@@ -71,6 +71,31 @@ tests:
       env:
       - default: /usr/local/ci-secrets/cincinnati-secrets-cincinnati-credentials/registry_credentials_docker.json
         name: CINCINNATI_TEST_CREDENTIALS_PATH
+      - default: /usr/local/ci-secrets/ci-image-sa-dockercfg/.dockercfg
+        name: CINCINNATI_CI_DOCKERCFG_PATH
+      - default: /usr/local/ci-secrets/public-ci-image-sa-dockercfg/.dockercfg
+        name: CINCINNATI_CI_PUBLIC_DOCKERCFG_PATH
+      - default: ${SHARED_DIR}/ci-image-sa-dockerjson
```
Comment (vrutkovs) [inline ci-operator/config/openshift/cincinnati/openshift-cincinnati-master.yaml:78]: Export filename here and add `${SHARED_DIR}` in `commands`?

Comment (steveeJ) [reply ci-operator/config/openshift/cincinnati/openshift-cincinnati-master.yaml]: Addressed, thanks!

**Review by vrutkovs — APPROVED** [review]
/lgtm

**Pull request merged**